tests/publiccloud/cloud_netconfig.pm

```diff
@@ -128,5 +128,9 @@ sub debug {
     }
 }
 
+sub test_flags {
+    return {fatal => 1, publiccloud_multi_module => 1};
```

Comment:
this means that if cloud_netconfig will have some bug we will not have coverage of suspending VM functionality . this tightly coupling of to independent test cases does not make sense to me . Can you elaborate why you decided that this two needs to run together ?